pom.xml

```diff
@@ -20,7 +20,7 @@
 
     <groupId>com.ibm.websphere.azure</groupId>
     <artifactId>azure.websphere-traditional.cluster</artifactId>
-    <version>1.4.1</version>
+    <version>1.4.2</version>
```

Comment:
Please change this to 1.3.35 and push another commit.

Reply:
Done with https://github.com/WASdev/azure.websphere-traditional.cluster/pull/170/commits/69ff1d171a1991f4dade0c671c5c9b620403e6e8.